### DISCUSSION TITLE
Sporty shoes Project doubts

### DISCUSSION THREAD

**Discussion by VikasAK47** [akash-coded/spring-framework#71] [Q&A]
Hello sir, after going through all the concepts its getting hard to implement all those concepts in one project. In class we used spring boot only to print messages, it would help if you had shown to implement All CRUD operations using Spring Boot. I have created all the packages as you asked , but I'm not able to understand where to use what, spring boot u showed to print message, crud you showed in xml configuration way ,its all getting mixed up for me.
Please help me understand how to implement All crud operation in today's class for at least one table.

**Comment by akash-coded** [comment]
@VikasAK47 I hope the last session gave you clarity on tying up all the loose ends and gave you a sense of direction on how to approach the project. Do let me know if you have any other doubts.

**Reply by akshaye98** [reply]
can you please upload the codes which was written on last day

**Reply by VikasAK47** [reply]
> @VikasAK47 I hope the last session gave you clarity on tying up all the loose ends and gave you a sense of direction on how to approach the project. Do let me know if you have any other doubts.

@akash-coded hello sir ,can you please Upload the project you had shown in class.

**Comment by VikasAK47** [comment]
@akash-coded yes sir I got an idea to start the project, ill let you know if i have any doubts.
Sir Please Upload the last days code
